include/AutomationPattern.h

```diff
@@ -38,6 +38,43 @@ class MidiTime;
 
 
 
+class AutomationNode
```

Comment:
Do we want to put this in its own file?

Reply:
We can do it, for now it's small enough that it doesn't really ask for a separate header in my opinion. But once we start adding stuff to it might be better to!